howso/client/base.py

```diff
@@ -4017,13 +4019,30 @@ def react_group(
         trainee_id : str
             The trainee id.
 
-        new_cases : list of list of list of object or list of DataFrame
-            Specify a **set** using a list of cases to compute the conviction of
-            groups of cases as shown in the following example.
+        case_indices: list of lists of tuples of {str, int}, optional
+            A list of lists of case indices tuples containing the session ID and
+            the session training indices that uniquely identify trained cases.
+            Each sublist defines a set of trained cases to react to. Only one of
+            ``case_indices``, ``conditions``, or ``new_cases`` may be specified.
+        conditions: a list of mappings, optional
```

Comment:
Also the type hint uses dict but this uses Mapping. Should be consistent
```suggestion
        conditions: list of Mapping, optional
```

Reply:
(same in Trainee version)